### ISSUE TITLE
async for log error

### ISSUE THREAD

**Issue by Taruu** [tiagocoutinho/linuxpy#16]
what did I expect:

```py
>>> from linuxpy.video.device import Device
>>> with Device.from_id(0) as camera:
...     async for frame in camera:
...         print(f"frame {len(frame)}")
frame 10224
frame 10304
frame 10224
frame 10136
```

what did I get:

```py
from linuxpy.video.device import Device
with Device.from_id(0) as camera:
    async for frame in camera:
        print(f"frame {len(frame)}")
        
start wait
start read
end read
end wait
frame 153600
start wait
start read
```
I couldn't find where to disable these logs.

**Comment by Taruu** [comment]
problem already solved.

https://github.com/tiagocoutinho/linuxpy/commit/597a55d42829bd3eb1f5b72f9546dc236a91d05d

The pip repository contains an unupdated package :/

**Comment by tiagocoutinho** [comment]
Thanks for reporting!
I updated package on pypi.
It should be fixed in 0.11.0
from you favorite python environment:
`pip install -U linuxpy`
